password.proto

```diff
@@ -0,0 +1,11 @@
+enum Algorithm {
+  INVALID = 1;
+  SCRYPT = 2;
+}
+
+message Password {
+  required Algorithm algorithm = 1;
+  required bytes salt = 2;
+  required float length = 3;
```

Comment:
`float`? are you sure?

Reply:
That's what a `Number` translates too, natively. IIRC.